src/AcmProgressTracker/AcmProgressTracker.tsx

```diff
@@ -0,0 +1,183 @@
+    'pending' = 'pending',
+    'unknown' = 'unknown',
+    'sleep' = 'sleep',
+    'empty' = 'empty',
+}
+
+const useStyles = makeStyles({
+    /* istanbul ignore next */
+    parentContainer: {
+        display: 'flex',
+        paddingTop: '10px',
+    },
+    popoverParentContainer: {
+        display: 'inline-grid',
+    },
+    popoverBody: {
+        display: 'flex',
+    },
+    stepContainer: {
+        display: 'inline-flex',
+        padding: '10px 0px 10px 0px',
+    },
+    text: { width: 'max-content' },
+    divider: {
+        padding: '0px 40px 0px 40px',
+        maxWidth: '180px',
+        maxHeight: '20px',
+    },
+    stepStatus: {
+        paddingLeft: '25px',
+    },
+    container: {
+        display: 'flex',
+    },
+    icon: {
+        width: '18px', // Progress size md is 18px
+    },
+    iconMargin: {
+        margin: '3px 2px 1px 2px',
+    },
+    button: {
+        padding: 0,
+        fontSize: 'inherit',
+    },
+})
+
+const divider = (isStacked: boolean) => {
+    const classes = useStyles()
+    if (isStacked) return undefined
+    return (
+        <svg className={classes.divider}>
+            <line
+                x1="0"
+                x2="100"
+                y1="10"
+                y2="10"
+                stroke="#D2D2D2" // --pf-global--palette--black-300
+                strokeWidth="2"
+                strokeLinecap="square"
+                strokeDasharray="1, 3"
+            />
+        </svg>
+    )
+}
+
+export function AcmProgressTracker(props: AcmProgressTrackerProps) {
+    const classes = useStyles()
+    const { viewWidth } = useViewport()
+    const isStacked = props.isStacked || viewWidth < 700
+
+    return (
+        <div>
+            <TextContent>
+                <Text component="h3">{props.Title}</Text>
+                <Text component="small">{props.Subtitle}</Text>
+            </TextContent>
+            <div>
+                <div className={isStacked ? classes.popoverParentContainer : classes.parentContainer}>
+                    {props.steps.map((step, index) => (
+                        <div key={index} className={classes.stepContainer}>
+                            <div>
+                                <InlineStatus type={step.statusType} status={step.statusText} popover={step.popover} />
+                                <TextContent>
+                                    <Text className={classes.stepStatus} component="small">
+                                        {step.statusSubtitle}
+                                    </Text>
+                                </TextContent>
+                            </div>
+                            {index < props.steps.length - 1 && divider(isStacked)}
+                        </div>
+                    ))}
+                </div>
+            </div>
+        </div>
+    )
+}
+
+export function InlineStatus(props: { type: StatusType; status: string | React.ReactNode; popover?: PopoverProps }) {
```

Comment:
Why do we need this copy of AcmInlineStatus?

Reply:
<img width="182" alt="Screen Shot 2021-05-13 at 10 17 51 AM" src="https://user-images.githubusercontent.com/21374229/118141635-912e4000-b3d7-11eb-94cd-459a20ffae62.png">

The component here will be, to a large extent, behaving a lot like our AcmInlineStatus, I thought it wouldn't make sense to re-build the wheel here...alas, since we are contributing this to PF I didn't think importing from ui-components, or using the Acm nomenclature made sense...so I pulled the code over into the same file and re-named it. 

This is definitely a decision that really could benefit from some discussion.

Reply:
Removed the copied code and now importing AcmInlineStatus